spec/hbw/features/bp_form/checkbox_icon_mock.yml

```
@@ -0,0 +1,124 @@
+get:
+  user:
+    - params: "email=c.johnson@example.com"
+      response:
+        data:
+          - id:         'demo'
+            firstName:  'Demo'
+            lastName:   'Demo'
+            url:        'http://bpm.example.com'
+            email:      'c.johnson@example.com'
+            pictureUrl: null
+        total: 1
+        start: 0
+        sort: 'id'
+        order: 'asc'
+        size: 1
+
+  # process definitions
+  process-definition:
+    - params: ""
+      response:
+        - &default_process-definition
+          id: "testProcess:469:222221"
+          key: "supportRequestProcess"
+          version: 469
+          name: "Test name"
+          description: null
+          tenantId: null
+          deploymentId: "222221"
+          resource: "testProcess.bpmn"
+          diagram: null
+          category: "http://bpmn.io/schema/bpmn"
+          suspended: false
+          versionTag: null
+          historyTimeToLive: null
+
+  # form definitions
+  task/111111/deployed-form:
+    - params: ""
+      response: |
+        form:
+          css_class: col-xs-12 col-sm-6 col-md-5 col-lg-4
+          fields:
+            - name: CustomIconCheckbox
+              type: checkbox
+              label: Checkbox label 1
+              unchecked:
+                class: far
+                name: circle
+              checked:
+                class: fas
+                name: square
```

Can't see any advantages of separating these options. Anyone who sets custom icons knows what style they should have. I mean, what the point of setting smth like
```
              unchecked:
                class: far
              checked:
                name: square
```
?

I can suggest this
```suggestion
              icon:
                checked: far circle
                unchecked: fas square
```